python/tests/scripts/a1_symbols.metta

```diff
@@ -1,11 +1,14 @@
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+; `match` searches for all expressions corresponding to
+; the given pattern and produces the output pattern.
+; It doesn't search in subexpressions.
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ; Some expressions to be matched
 ((leaf1 leaf2) leaf3)
 (((leaf0 leaf1) leaf2) leaf3)
-; This one will not be matched
+; This one will not be matched because it's a subexpression
```

Comment:
"It's" might be not clear here. Maybe, we need to be explicit here, e.g. "This one contains ((leaf1 leaf2) leaf3) as a subexpression and thus will not be matched"